### PR TITLE
collapse hide inlines on admin change form

### PR DESCRIPTION
**Pull request by s-block**

### CODE REVIEW

**Comment by s-block** [comment]
screen shot 2016-09-14 at 14 51 27

screen shot 2016-09-14 at 14 51 53

File: dashboard/assets/admin/dashboard/styles/custom.css [hunk 1]
```diff
@@ -435,3 +435,12 @@ th.required::before {
 .form-row a[rel="external"]:after {
     background-position: right 3px;
 }
+
+div.inline-group div.inline-related fieldset.module table tr.toggle-collapsed-inlines td {
+    cursor: pointer;
+    height: 10px;
+    background-color: #f8f8f8;
+    border-bottom: 1px solid #eee;
+    text-align: center;
```
Comment (cliffxuan) [inline dashboard/assets/admin/dashboard/styles/custom.css]: might be better to align to the left. it tends to be more noticeable by most of the people.

File: dashboard/assets/admin/dashboard/js/CollapseInline.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      var $tbody = $(this);
+      var $trs = $tbody.find('tr').not('.add-row').not('.empty-form');
+      if ($trs.length > SHOW_REMAINING) {
+        var colspan = $trs.first().children('td').length;
+        var $expandTr = $('<tr class="toggle-collapsed-inlines collapsed" />');
+        var $expandTd = $('<td colspan="' + colspan + '" />');
+        $expandTr.append($expandTd);
+        $tbody.prepend($expandTr);
+
+        function toggleRows ($rows) {
```
Comment (cliffxuan) [inline dashboard/assets/admin/dashboard/js/CollapseInline.js]: it's probably better to show tops 3 rows rather than bottom 3 rows.

**Comment by s-block** [comment]
I tried it on the right and it wasn't great. I have made the bar green and the text bold

screen shot 2016-09-14 at 15 28 01

**Comment by cliffxuan** [comment]
👍

**Pull request merged**